aws/resource_aws_s3_bucket_policy_test.go

```diff
@@ -17,16 +17,22 @@ func TestAccAWSS3BucketPolicy_basic(t *testing.T) {
 	partition := testAccGetPartition()
 
 	expectedPolicyText := fmt.Sprintf(`{
-	"Version": "2012-10-17",
-	"Statement": [{
-		"Sid": "",
-		"Effect": "Allow",
-		"Principal": {"AWS":"*"},
-		"Action": "s3:*",
-		"Resource": ["arn:%s:s3:::%s/*","arn:%s:s3:::%s"]
-	}]
-}
-`, partition, name, partition, name)
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Sid": "",
+      "Effect": "Allow",
+      "Principal": {
+        "AWS": "*"
+      },
+      "Action": "s3:*",
+      "Resource": [
+        "arn:%s:s3:::%s/*",
+        "arn:%s:s3:::%s"
+      ]
+    }
+  ]
+}`, partition, name, partition, name)
```

Comment:
This could use `%[x]s` to remove repetition, here and in similar cases

Reply:
Fixed!